src/modules/dashboard/components/navbar/Navbar.jsx

```diff
@@ -0,0 +1,41 @@
+import React from 'react';
+import { Link } from "react-router-dom"
+import "./navbar.styles.css"
+import { ReactComponent as BellLogo} from "../../../../assets/svg/bell.svg"
+import UserImage from "../../../../assets/image/mheader.png"
+
+
+const Navbar = () => {
+  return (
+    <div className="navbar">
+      <ul className="header-nav">
+        <li className="header-nav-item">
+          <Link to="#" className="active"> Requests</Link>
+          <span className="span-active">24</span>
+        </li>
+        <li className="header-nav-item">
+          <Link to="#" className=""> Suggestions</Link>
+          <span className="">24</span>
+        </li>
+        <li className="header-nav-item">
+          <Link to="#" className=""> Matches</Link>
+          <span className="">24</span>
+        </li>
+      </ul>
+      <div className="profile-nav">
+        <div className="bell">
+          <BellLogo/>
```

Comment:
Why are these svgs rendered as React components and not used as images?

Reply:
I rendered it as a component just in case we implement the notification alert (the blue dot), it'll be easier to set if it's active or not.

Reply:
Got it